manifests/tftp/netboot.pp

```diff
@@ -34,6 +34,8 @@
       file { "${root}/grub2/shim.efi":
         ensure => file,
         source => "/boot/efi/EFI/${grub_efi_path}/shim.efi",
+        mode  => '0644',
```

Comment:
Could you align the arrows? That should make puppet-lint happy and allow the build to pass